packages/timeline-state-resolver/src/devices/atem.ts

```diff
@@ -248,8 +255,12 @@ export class AtemDevice extends DeviceWithState<DeviceState, DeviceOptionsAtemIn
 								const me = AtemStateUtil.getMixEffect(deviceState, mapping.index)
 								const atemObj = tlObject as any as TimelineObjAtemME
 								const atemObjKeyers = atemObj.content.me.upstreamKeyers
+								const transition = atemObj.content.me.transition
 
 								deepExtend(me, _.omit(atemObj.content.me, 'upstreamKeyers'))
+								if (this._isAssignableToNextStyle(transition)) {
+									me.transitionProperties.nextStyle = transition as number as ConnectionEnums.TransitionStyle
```

Comment:
Purely by looking at this, I assume that the `nextStyle` is somehow persisted through restart and that's what makes this work?

Also, would be nice to have some test that ensures that `nextStyle` is sat if it is valid, so we don't accidentally delete this again :)

Reply:
I added some tests